bassist/bassist/flavor/db_pg.py

```diff
@@ -18,12 +18,12 @@ class FlavorDBException(Exception):
 class DB(object):
     def __init__(self, path=None):
         print "Hello db_pg"
```

Comment:
This is a debugging statement; please either use the log command, or remove this.